### PR TITLE
project not needed for build

### PR DESCRIPTION
**Pull request by itayd**
- no need for resources_paths from project - everything in a directory is built
- there is no need for project reference in build, this makes it completely decoupled from a project

### CODE REVIEW

File: cmd/ak/cmd/builds/upload.go [hunk 1]
```diff
@@ -7,13 +7,9 @@ import (
...
 	"go.autokitteh.dev/autokitteh/cmd/ak/common"
-	"go.autokitteh.dev/autokitteh/internal/kittehs"
-	"go.autokitteh.dev/autokitteh/internal/resolver"
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
-var project string
-
 var uploadCmd = common.StandardCommand(&cobra.Command{
 	Use:     "upload <build file path> <--project=...>",
```
Comment (daabr) [inline cmd/ak/cmd/builds/upload.go]:
```suggestion
	Use:     "upload <build file path>",
```

File: cmd/ak/cmd/builds/upload.go [hunk 2]
```diff
@@ -26,17 +22,7 @@ var uploadCmd = common.StandardCommand(&cobra.Command{
...
-		if err != nil {
-			return err
-		}
-		if p == nil {
-			err = fmt.Errorf("project %q not found", project)
-			return common.NewExitCodeError(common.NotFoundExitCode, err)
-		}
-
-		b, err := sdktypes.BuildFromProto(&sdktypes.BuildPB{ProjectId: p.ToProto().ProjectId})
+		b, err := sdktypes.BuildFromProto(&sdktypes.BuildPB{})
```
Comment (daabr) [inline cmd/ak/cmd/builds/upload.go]: Should we change `builds().Save()` to not require a build object as a parameter, given that it's empty by definition?

File: backend/internal/projects/projects.go [hunk 3]
```diff
@@ -97,9 +91,7 @@ func (ps *Projects) Build(ctx context.Context, projectID sdktypes.ProjectID) (sd
 		return nil, err
 	}
 
-	build := kittehs.Must1(sdktypes.BuildFromProto(&sdktypes.BuildPB{
-		ProjectId: projectID.String(),
-	}))
+	build := kittehs.Must1(sdktypes.BuildFromProto(&sdktypes.BuildPB{}))
```
Comment (daabr) [inline backend/internal/projects/projects.go]: Doesn't it make more sense to create a new object directly, without using an empty PB and muting it's impossible translation error?

File: systest/testdata/builds/build.txtar [hunk 4]
```diff
@@ -0,0 +1,217 @@
+ak builds create --path hiss.star
```
Comment (daabr) [inline systest/testdata/builds/build.txtar:2]:
```suggestion
# Negative test: build failure due to error in code.
ak builds create --path hiss.star
```

File: systest/testdata/builds/build.txtar [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+output equals file hiss_error.txt
+return code == 1
+
+ak builds create --path single.star -o single.akb -d
```
Comment (daabr) [inline systest/testdata/builds/build.txtar]:
```suggestion
ak builds create --path single.star -o single.akb --describe
```

File: systest/testdata/builds/build.txtar [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+output equals_json file single_output.json
+return code == 0
+
+ak builds create --dir multi -o multi.akb -d
```
Comment (daabr) [inline systest/testdata/builds/build.txtar]:
```suggestion
ak builds create --dir multi -o multi.akb --describe
```

**Review by daabr — COMMENTED** [review]
Mostly nits, plus 1 small fix and a question.

File: cmd/ak/cmd/builds/upload.go (hunk 2 above)
Comment (itayd) [reply cmd/ak/cmd/builds/upload.go]: we might want to add stuff in the future. i think we should leave it as is.

File: backend/internal/projects/projects.go (hunk 3 above)
Comment (itayd) [reply backend/internal/projects/projects.go]: yeah it's silly. going to add an sdktypes.NewBuild() function.

File: cmd/ak/cmd/builds/upload.go (hunk 2 above)
Comment (itayd) [reply cmd/ak/cmd/builds/upload.go]: used NewBuild

**Review by daabr — APPROVED** [review]

**Pull request merged**